ecl/baremetal/v2/server.py

```diff
@@ -254,6 +254,17 @@ def get_management_console(self, session, server_id):
         self._translate_response(resp, has_body=True)
         return self
 
+    def reset_bmc(self, session, server_id):
+        uri = self.base_path % server_id
+        body = {"bmc-reset": None}
```

Comment:
type (WARM / COLD)が指定できる必要があるのでは？
引数に typeを追加して、
media_detachメソッドを真似ると、bodyは以下のように作成すべきかと。
```suggestion
        body = {
            "bmc-reset": {
                "type": type
            }
        }
```

Reply:
ありがとうございます。修正いたしました。